reservaciones/src/screens/Home.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { View, Text } from "react-native";
+
+export const Home = ({navigation}) => {
+    return(
+        <View>
+            <Text>Home page</Text>
+        </View>
+    )
+}
```

Comment:
Check the indentation of your file (a spacing of 2 is recommended).

Add a blank line at the end.